.github/ci_script/combine_log.py

```diff
@@ -0,0 +1,45 @@
+import time
+import sys
+import os
+'''
+  Get info.
```

Comment:
Get what info? Comments more about this file.

Reply:
Done